microsetta_public_api/server.py

```diff
@@ -37,7 +37,7 @@ def build_app(resource_updates=None):
     else:
         with open(config_fp) as fp:
             server_config = json.load(fp)
-        resource_config = config['resources']
+        resource_config = server_config.get('resources', None)
```

Comment:
I really did not look close at when putting this change in... sorry about that

Reply:
No worries. It should have been caught by tests, but the way things were refactored made it so tests weren't covering the config from a JSON case any more.